GNUmakefile

```diff
@@ -24,7 +24,9 @@ ifeq ($(UNAME_P),i386)
 	endif
 else
 	ifeq ($(UNAME_P),AMD64)
```

Comment:
Think we saw that `uname -p` could give `Unknown` under WSL: :)

Reply:
Got myself a VM in Fusion Player with Windows 11 Pro N and WSL 2. In Ubuntu (20.04 and 22.04) `uname -p` reports `x86_64` but in Debian 11 it reports `unknown`.

Reply:
Added a table in the description we can add info to

Reply:
`go tool dist list` shows all supported platforms (which I believe is the combination of `GOOS` + `GOARCH`):

```
$ go tool dist list

aix/ppc64
android/386
android/amd64
android/arm
android/arm64
darwin/amd64
darwin/arm64
dragonfly/amd64
freebsd/386
freebsd/amd64
freebsd/arm
freebsd/arm64
illumos/amd64
ios/amd64
ios/arm64
js/wasm
linux/386
linux/amd64
linux/arm
linux/arm64
linux/loong64
linux/mips
linux/mips64
linux/mips64le
linux/mipsle
linux/ppc64
linux/ppc64le
linux/riscv64
linux/s390x
netbsd/386
netbsd/amd64
netbsd/arm
netbsd/arm64
openbsd/386
openbsd/amd64
openbsd/arm
openbsd/arm64
openbsd/mips64
plan9/386
plan9/amd64
plan9/arm
solaris/amd64
windows/386
windows/amd64
windows/arm
windows/arm64
```

So for instance there seems to be no support for 32 bit at all on macOS.